bugzooka/integrations/inference.py

```diff
@@ -132,6 +134,21 @@ def analyze_log(product: str, product_config: dict, error_summary: str) -> str:
         except KeyError:
             formatted_content = prompt_config["user"].format(summary=error_summary)
 
+        # Optionally augment with RAG context when RAG_IMAGE is set
+        rag_image = os.getenv("RAG_IMAGE")
+        rag_query = os.getenv("RAG_QUERY", "Summarize the the rag database")
```

Comment:
Here instead of saying to summarize, we need to pass the error summary as a query to RAG so that relevant content if fetched as context.